src/object.js

```diff
@@ -19,6 +19,16 @@ const noCap = _.convert({ cap: false })
 export const singleObject = _.curry((key, value) => ({ [key]: value }))
 export const singleObjectR = _.flip(singleObject)
 
+// {a:{name:'foo'},b:{name:'bar'}} => [fn('foo'), fn('bar')]
+export const extract = (key, from, run) => {
```

Comment:
It seems like this method should just handle the extraction and not the transformation - like a `_.get` that supports getting by some criteria. That would allow it to compose better with other methods.

Also, the implementation here can likely be simplified using `_.pickBy`

I'm not 100% sold on the name - it doesn't seem like it communicates that it will get a list of values - perhaps something like `getAll`, but that doesn't convey the nesting. `getAllNested` could work but it is a bit verbose. Definitely open to suggestions